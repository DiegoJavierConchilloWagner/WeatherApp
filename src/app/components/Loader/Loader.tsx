import { StyledLoader } from './styled';

export const Loader = () => {
	return (
		<StyledLoader className='lds-ellipsis'>
			<div />
			<div />
			<div />
			<div />
		</StyledLoader>
	);
};
