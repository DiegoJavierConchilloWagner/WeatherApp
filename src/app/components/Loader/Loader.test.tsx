import renderWithThemeAndProvider from '@common/utils/renderWhitThemeAndProvider';
import { Loader } from './Loader';

describe('Example Loader', () => {
	it('should render correcty', () => {
		const wrapper = renderWithThemeAndProvider(<Loader />);

		expect(wrapper).toMatchSnapshot();
	});
});
