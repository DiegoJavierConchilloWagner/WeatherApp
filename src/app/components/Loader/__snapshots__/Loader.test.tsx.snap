// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example Loader should render correcty 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": .c0 {
  display: inline-block;
  position: relative;
  width: 80px;
  height: 80px;
}

.c0 div {
  position: absolute;
  top: 33px;
  width: 13px;
  height: 13px;
  border-radius: 50%;
  background: #fff;
  -webkit-animation-timing-function: cubic-bezier(0,1,1,0);
  animation-timing-function: cubic-bezier(0,1,1,0);
}

.c0 div:nth-child(1) {
  left: 8px;
  -webkit-animation: lds-ellipsis1 0.6s infinite;
  animation: lds-ellipsis1 0.6s infinite;
}

.c0 div:nth-child(2) {
  left: 8px;
  -webkit-animation: lds-ellipsis2 0.6s infinite;
  animation: lds-ellipsis2 0.6s infinite;
}

.c0 div:nth-child(3) {
  left: 32px;
  -webkit-animation: lds-ellipsis2 0.6s infinite;
  animation: lds-ellipsis2 0.6s infinite;
}

.c0 div:nth-child(4) {
  left: 56px;
  -webkit-animation: lds-ellipsis3 0.6s infinite;
  animation: lds-ellipsis3 0.6s infinite;
}

<body>
    <div>
      <div
        class="c0 lds-ellipsis"
      >
        <div />
        <div />
        <div />
        <div />
      </div>
    </div>
  </body>,
  "container": .c0 {
  display: inline-block;
  position: relative;
  width: 80px;
  height: 80px;
}

.c0 div {
  position: absolute;
  top: 33px;
  width: 13px;
  height: 13px;
  border-radius: 50%;
  background: #fff;
  -webkit-animation-timing-function: cubic-bezier(0,1,1,0);
  animation-timing-function: cubic-bezier(0,1,1,0);
}

.c0 div:nth-child(1) {
  left: 8px;
  -webkit-animation: lds-ellipsis1 0.6s infinite;
  animation: lds-ellipsis1 0.6s infinite;
}

.c0 div:nth-child(2) {
  left: 8px;
  -webkit-animation: lds-ellipsis2 0.6s infinite;
  animation: lds-ellipsis2 0.6s infinite;
}

.c0 div:nth-child(3) {
  left: 32px;
  -webkit-animation: lds-ellipsis2 0.6s infinite;
  animation: lds-ellipsis2 0.6s infinite;
}

.c0 div:nth-child(4) {
  left: 56px;
  -webkit-animation: lds-ellipsis3 0.6s infinite;
  animation: lds-ellipsis3 0.6s infinite;
}

<div>
    <div
      class="c0 lds-ellipsis"
    >
      <div />
      <div />
      <div />
      <div />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
